### ISSUE TITLE
문제 Yolov5구현

### ISSUE THREAD

**Issue by Batwan01** [ultralytics/yolov5#13339]

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Batwan01, thank you for reaching out about YOLOv5 🚀! This is an automated response, and an Ultralytics engineer will assist you soon. 

To get started, please check out our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5/) for helpful guides on [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/) and more advanced topics like [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution/).

If this is a 🐛 Bug Report, we kindly ask you to provide a **minimum reproducible example** to help us debug the issue effectively.

In the case of a custom training ❓ Question, please offer as much information as possible, including dataset examples and training logs. Make sure you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

## Requirements

Ensure you are using [**Python>=3.8.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Start by running:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

## Environments

YOLOv5 can be run in verified environments with dependencies, including [CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. This includes [training](https://github.com/ultralytics/yolov5/blob/master/train.py), [validation](https://github.com/ultralytics/yolov5/blob/master/val.py), [inference](https://github.com/ultralytics/yolov5/blob/master/detect.py), [export](https://github.com/ultralytics/yolov5/blob/master/export.py), and [benchmarks](https://github.com/ultralytics/yolov5/blob/master/benchmarks.py) on various operating systems.

## Introducing YOLOv8 🚀

We're thrilled to announce [YOLOv8](https://github.com/ultralytics/ultralytics) - our latest object detection model! YOLOv8 offers cutting-edge features for object detection, image segmentation, and classification. Explore the [YOLOv8 Docs](https://docs.ultralytics.com/) and get started:
```bash
pip install ultralytics
```

Please let us know any additional details to assist you better! 😊

**Comment by pderrenger** [comment]
@Batwan01 please provide more details about the issue you're experiencing with YOLOv5, so we can assist you better.
